examples/showcase-anime-UI/index.html

```diff
@@ -8,18 +8,19 @@
   </head>
   <body>
     <a-scene>
```

Comment:
[http://localhost:9000/examples/showcase-anime-UI/](http://localhost:9000/examples/showcase-anime-UI/) loads in Firefox but doesn't load in Chrome (does load on `dev`):

<img width="1363" alt="screenshot 2016-02-17 02 57 39" src="https://cloud.githubusercontent.com/assets/203725/13107850/970bc688-d524-11e5-95a4-331319fba6a8.png">
![screenshot 2016-02-17 02 57 41](https://cloud.githubusercontent.com/assets/203725/13107837/7d4d839e-d524-11e5-801d-908af6750d02.png)
![screenshot 2016-02-17 03 00 21](https://cloud.githubusercontent.com/assets/203725/13107836/7d4b4124-d524-11e5-90dc-db446b77388b.png)